deduplication/src/main/java/gov/cdc/nbs/deduplication/matching/mapper/LinkRequestMapper.java

```diff
@@ -36,9 +33,9 @@ public LinkRequest map(PersonMatchRequest request) {
       birthDate = formatter.format(request.personDto().birthTime());
     }
 
-    String race = null;
+    List<String> race = null;
     if (request.races() != null && !request.races().isEmpty()) {
-      race = request.races().get(0).raceCategoryCd();
+      race = Collections.singletonList(request.races().get(0).raceCategoryCd());
```

Comment:
Suggestion: Instead of now only returning the first race `.get(0)`, should we send all available races since Record Linker now supports that?